docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -114,6 +114,31 @@ autonomy deploy build keys.json -ltm
 
 Builds a service deployment using the keys stored in the file `keys.json` and applying environment variables to the service configuration file. The deployment will be generated by default for as many agents as keys are stored in `keys.json`. By default, the command searches for the file `keys.json`, if no file name is provided.
 
+### Private key security
+
+When building deployments, you can use password protected privates keys (eg. generated using `autonomy generate-key LEDGER --password PASSWORD`) to avoid exposing them. 
+
+#### Docker Compose
+
+In a `docker-compose` deployment you can just export `OPEN_AUTONOMY_PRIVATE_KEY_PASSWORD` environment variable before running the deployment like
+
+```bash
+OPEN_AUTONOMY_PRIVATE_KEY_PASSWORD=PASSWORD autonomy deploy run --build-dir PATH_TO_BUILD_DIR
+```
+
+or 
+
+```bash
+OPEN_AUTONOMY_PRIVATE_KEY_PASSWORD=PASSWORD docker-compose -f PATH_TO_BUILD_DIR/docker-compose.yaml up
```

Comment:
```suggestion
OPEN_AUTONOMY_PRIVATE_KEY_PASSWORD=PASSWORD
docker-compose -f PATH_TO_BUILD_DIR/docker-compose.yaml up
```

Reply:
Why do we have docker-compose commands here? they are not used anywhere in the docs, I think...

Reply:
Since the deployment builds are in form of docker-compose, someone might wanna use docker-compose instead of  autonomy to deploy the service. So this is here just in case